### DISCUSSION TITLE
can one use slint for a project distributed as MIT license?

### DISCUSSION THREAD

**Discussion by soloturn** [slint-ui/slint#2226] [Q&A] [answered]
can one use slint for a project using MIT license, like https://github.com/mikedilger/gossip, which currently uses egui?

**Comment by bjorn** [comment, marked as answer]
Not really qualified to answer, but I think you have a number of options:

* By dual-licensing your MIT code as MIT + GPLv3. As far as I know MIT allows this, but it means the restrictions of GPLv3 apply (which only exist to keep your program free, though).
* Using the terms of the [Ambassador Program](https://slint-ui.com/ambassador-program.html) instead could allow you to keep your project purely MIT. It doesn't sound like a hard one, especially for an open source project.
* Buying a Slint license.

**Comment by ogoffart** [comment]
Hi, 

We try to answer this question there https://github.com/slint-ui/slint/blob/master/FAQ.md#can-i-license-my-code-under-an-open-source-license-more-permissive-than-gpl

The MIT license is compatible with the GPL license, so you can mix them.
Someone that redistributes a binary of the program would need to respect one of the license of Slint: either the GPL, or get a proprietary license (of which there is a free tier)
